docs/DeveloperGuide.md

```diff
@@ -403,106 +403,6 @@ The following activity diagram summarizes what happens when a user executes a ne
     - Pros: Attendance can be modified easily.
     - Cons: Implementation is relatively complicated and require more exception handling.
 
-### \[Proposed\] Undo/redo feature
```

Comment:
Thank you for removing unnecessary parts

Reply:
👍